src/services/config.js

```diff
@@ -156,13 +166,13 @@ class Config {
     if (!fs.existsSync(this.filePath)) {
       return configData;
     }
-
```

Comment:
noooooooooooo

Reply:
omg it's not even a _good_ use of whitespace